lib/chef/resource/windows_package.rb

```diff
@@ -26,13 +26,12 @@ class Resource
     class WindowsPackage < Chef::Resource::Package
       include Chef::Mixin::Uris
 
-      provides :package, os: "windows"
       provides :windows_package, os: "windows"
+      provides :package, os: "windows"
```

Comment:
`os: "freebsd"` was removed above this, so if they're both correct, I'm curious why. =)